tiledb/sm/rest/rest_client.h

```diff
@@ -128,6 +128,29 @@ class RestClient {
    */
   void delete_array_from_rest(const URI& uri);
 
+  /**
+   * Deletes the fragments written between the given timestamps from the array
+   * at the given URI from the REST server.
+   *
+   * #TODO Implement API endpoint on TileDBCloud.
+   * @param uri Array URI to delete
```

Comment:
NIT: The `Array URI to delete fragments from` that you have in the next method sounds better here too (less scary :D )